python/triton/tools/disasm.py

```diff
@@ -78,9 +76,7 @@ def get_sass(cubin_asm, fun=None):
 
 
 def extract(file_path, fun):
-    cuobjdump, _ = path_to_cuobjdump()
-    nvdisasm, _ = path_to_nvdisasm()
-    os.environ["NVDISASM_PATH"] = nvdisasm
+    cuobjdump = "cuobjdump"
```

Comment:
ooc how does this work?  How do we know that the cuobjdump we downloaded is in the system's PATH?

Reply:
that's a good point, this was assuming cuobjdump is on PATH. 
Changed it to use the same method as other nv binaries.